### DISCUSSION TITLE
allergies filter

### DISCUSSION THREAD

**Discussion by snickers2k** [mealie-recipes/mealie#3265] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

people can have allergies and can not eat everything

### Please provide a concise description of the feature that would resolve your issue.

would be great to have a **filter for allergies**. **so a filter to exclude/avoid certain ingredients.**

**Comment by Kuchenpirat** [comment]
You should already be able to do this rudimentary with categories or tags. 

Its quite manual, but you could for e.g. add the category "gluten free" to a recipe and only show recipes that have that category. It's the reverse of what your want and only shows recipes containing the category instead of excluding the ones containing the category but it would work. 

This does not match your request exactly but could be a first step if your household does include people with some extended dietary restrictions.

**Comment by snickers2k** [comment]
yes, i know. and for the own household, it surely is some workaround.
but adding by hand takes lot of time and needs preperation.

maybe this needs a data-set of course, to know "yogurt = milk-protein" , so maybe this could help:
https://db.comparedatabase.org

https://fdc.nal.usda.gov/fdc-app.html#/
